Main/Source/nonhuman.cpp

```diff
@@ -922,6 +922,29 @@ truth spider::SpecialBiteEffect(character* Char, v2, int, int, truth BlockedByAr
     return false;
 }
 
+truth vampirebat::SpecialBiteEffect(character* Victim, v2 HitPos, int BodyPartIndex, int Direction, truth BlockedByArmour, truth Critical, int DoneDamage)
```

Comment:
I don't know hot to offhand, but is there any way you could make the `vampirebat::SpecialBiteEffect` just call the `vampire::SpecialBiteEffect`?  Having all this code twice is suboptimal.